README.md

```diff
@@ -32,8 +12,8 @@ search for can be filtered by using a search query.
 |Time to Answer (Discussions Only) | The time from creation to an answer.|
 |Time in Label | The duration from label application to removal, requires LABELS_TO_MEASURE env variable.|
 
-*For pull requests, these metrics exclude the time the PR was in draft mode.
-*For Issue and pull requests, issue/pull request author's own comments and comments by bots are excluded.
+* For pull requests, these metrics exclude the time the PR was in draft mode.
```

Comment:
Is this a typo in the item enumeration?

Reply:
these are intended to be footnotes

Reply:
I've reverted the changes to "Example use cases."  ( https://github.com/github/issue-metrics/pull/139/commits/659c287446fda32d1d830c23deaf678353c93533 )